lib/powerpoint.js

```diff
@@ -167,20 +277,17 @@ class Powerpoint extends EventEmitter2 {
   _createSlides (imageFiles, pptFile, done) {
     var start = this.nowInMillis()
     var pptx = officegen('pptx')
+    var d = this.options.dimensions
+    // https://github.com/Ziv-Barber/officegen/issues/112
```

Comment:
It would be helpful to offer a brief explanation of your intent here.  It's not obvious from looking at the issue why you linked it, or why setSlideSize option is called